### PR TITLE
[go/it] Add first italian translation of go language

### PR DESCRIPTION
**Pull request by neslinesli93**

### CODE REVIEW

File: it-it/go-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,454 @@
+---
+name: Go
+category: language
```
Comment (vendethiel) [inline it-it/go-it.html.markdown]: that shouldn't be necessary :)

**Comment by neslinesli93** [comment]
Is there any reason this hasn't been merged yet?

**Comment by vendethiel** [comment]
yes, my comment

**Comment by vendethiel** [comment]
Thanks for your contribution. I fixed the document myself :)

@adambard Finally under 100PRs...

**Comment by adambard** [comment]
@vendethiel Great job!

**Pull request merged**